shared/src/native-src/pal.h

```diff
@@ -159,16 +173,18 @@ inline WSTRING GetCurrentProcessCommandLine()
     std::string name;
     std::stringstream tokens(cmdline);
     std::string tmp;
+    int i = 0;
```

Comment:
I'm not sure you still need this `i` variable

Reply:
👍 Missed in the refactoring

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/5564/commits/594e711959b590150fd6f1f1aeb2f7c222b239b8